W-10935528 | Add reader property in FlatFile's config file (#141)
mulesoft/docs-dataweave: modules/ROOT/pages/_partials/dataweave-formats-flatfile-config.adoc

certain single-byte character encodings
(so not with UTF-8 or any multibyte format).

| `allowLenientWithBinaryNotEndElement` | `Boolean` | `false` |

> If the schema contains elements of type *Binary* or *Packed* the *lenient* option will not allow short records, regardless of the last element

Reviewer: ```suggestion
When the schema contains elements of type Binary or Packed, the `lenient` option does not allow short records, regardless of the last element.
```

> format type. When you set this property to `true`, the validation will apply only in cases where the record ends with type `Binary` or `Packed`.

Reviewer: ```suggestion
format type. When you set this property to `true`, the validation applies only to cases in which the record ends with type Binary or Packed.
```
